patch/0042-ARM64-PCI-Allow-userspace-to-mmap-PCI-resources.patch

```diff
@@ -0,0 +1,59 @@
+From b5a750c95de583df71ab719ac9ae455bd6cfe7cc Mon Sep 17 00:00:00 2001
+From: gilt <gilt@marvell.com>
+Date: Mon, 26 Aug 2019 13:43:39 +0300
+Subject: [PATCH 4/6] ARM64/PCI: Allow userspace to mmap PCI resources
+
+Change-Id: I87ef6492e4956c77c75470ec010a4f11a9a8bdaf
```

Comment:
is this back port? if yes, what is original commit id in linux kernel tree?

Reply:
No, this is not a backport, For arm64 this change is been circulated in the kernel forum and still not taken.
https://lore.kernel.org/patchwork/patch/770393/